### PR TITLE
📝(git) add guideline for the commit message of a new release

### PR DESCRIPTION
**Pull request by carofun**
**Purpose**

Structuring commit messages in case of a new release

**Proposal**

Adding systematically to the body all the bullet point of the changelog file that are concerned by this new release

### CODE REVIEW

File: git.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ Whatever the language you are using on a FUN project, cooking a new release \(_e
 
 3. Update the project's `Changelog` following the [keepachangelog](https://keepachangelog.com/en/0.3.0/) recommandations,
 
-4. Commit your changes with a standard message title like: `Bump release to 4.18.1`,
+4. Commit your changes with a structured message, add a title including the name of the release like: `Bump release to 4.18.1`, add to the body all the changes from the changelog concerned by this release,
```
Comment (sampaccoud) [inline git.md:146]: maybe paste here an example of release commit?

Comment (carofun) [reply git.md]: I've added an exemple.

File: git.md [hunk 2]
```diff
@@ -143,7 +143,18 @@ Whatever the language you are using on a FUN project, cooking a new release \(_e
...
+
+```
+Added :
+
+ - Implement base CLI commands (list, extract, fetch & push) for supported
+ backends
+ - Support for ElasticSearch database backen
+ - Support for LDP storage backend
+ - Support for FS storage backend
+ - Parse (gzipped) tracking logs in GELF format
```
Comment (sampaccoud) [inline git.md:156]:
```suggestion
🔖(minor) Bump release to 4.18.1

Added :

 - Implement base CLI commands (list, extract, fetch & push) for supported
 backends
 - Support for ElasticSearch database backen
 
 Changed:
 - Replace LDP storage backend by FS storage backend
```

File: git.md [hunk 3]
```diff
@@ -143,7 +143,18 @@ Whatever the language you are using on a FUN project, cooking a new release \(_e
 
 3. Update the project's `Changelog` following the [keepachangelog](https://keepachangelog.com/en/0.3.0/) recommandations,
 
-4. Commit your changes with a standard message title like: `Bump release to 4.18.1`,
+4. Commit your changes with a structured message, add a title including the name of the release like: `Bump release to 4.18.1`, add to the body all the changes from the changelog concerned by this release, like :
```
Comment (sampaccoud) [inline git.md:146]:
```suggestion
4. Commit your changes with a structured message:
- add a title including the name of the release like and respecting the above described format using the release gitmoji,
- paste in the body all the changes from the changelog concerned by this release, removing only the markdown tags and making sure that lines are shorter than 74 characters to satisfy gitlint.
```

Comment (carofun) [reply git.md]: this is changed :-)

File: git.md (hunk 2 above)
Comment (carofun) [reply git.md]: Your exemple has been added.

File: git.md [hunk 4]
```diff
@@ -143,7 +143,23 @@ Whatever the language you are using on a FUN project, cooking a new release \(_e
 
 3. Update the project's `Changelog` following the [keepachangelog](https://keepachangelog.com/en/0.3.0/) recommandations,
 
-4. Commit your changes with a standard message title like: `Bump release to 4.18.1`,
+4. Commit your changes with a structured message:
+
+- add a title including the name of the release like and respecting the above described format using the release gitmoji,
```
Comment (jmaupetit) [inline git.md:148]:
```suggestion
- add a title including the version of the release and respecting the above described format using the 🔖 release emoji,
```

File: git.md [hunk 5]
```diff
@@ -143,7 +143,23 @@ Whatever the language you are using on a FUN project, cooking a new release \(_e
...
+
```
Comment (jmaupetit) [inline git.md:149]:
```suggestion
```

File: git.md [hunk 6]
```diff
@@ -143,7 +143,23 @@ Whatever the language you are using on a FUN project, cooking a new release \(_e
...
+- paste in the body all the changes from the changelog concerned by this release, removing only the markdown tags and making sure that lines are shorter than 74 characters to satisfy gitlint.
```
Comment (jmaupetit) [inline git.md:150]:
```suggestion
- paste in the body all changes from the changelog concerned by this release, removing only the markdown tags and making sure that lines are shorter than 74 characters.
```

File: git.md [hunk 7]
```diff
@@ -143,7 +143,23 @@ Whatever the language you are using on a FUN project, cooking a new release \(_e
...
+
+```
+🔖(minor) Bump release to 4.18.0
+
+Added :
+ - Implement base CLI commands (list, extract, fetch & push) for supported
+ backends
+ - Support for ElasticSearch database backen
```
Comment (jmaupetit) [inline git.md:158]:
```suggestion
 - Support for ElasticSearch database backend
```

File: git.md [hunk 8]
```diff
@@ -143,7 +143,23 @@ Whatever the language you are using on a FUN project, cooking a new release \(_e
...
+4. Commit your changes with a structured message:
+
+- add a title including the name of the release like and respecting the above described format using the release gitmoji,
+
+- paste in the body all the changes from the changelog concerned by this release, removing only the markdown tags and making sure that lines are shorter than 74 characters to satisfy gitlint.
+
+```
+🔖(minor) Bump release to 4.18.0
+
+Added :
```
Comment (jmaupetit) [inline git.md:155]:
```suggestion
Added :

```

File: git.md [hunk 9]
```diff
@@ -143,7 +143,23 @@ Whatever the language you are using on a FUN project, cooking a new release \(_e
...
+ - Implement base CLI commands (list, extract, fetch & push) for supported
+ backends
+ - Support for ElasticSearch database backen
+ 
+ Changed:
```
Comment (jmaupetit) [inline git.md:160]:
```suggestion
 Changed:
 
```

**Review by jmaupetit — COMMENTED** [review]
Yay, thank you for taking time to polish this part :pray:

File: git.md (hunk 4 above)
Comment (carofun) [reply git.md]: Changes have been done.

File: git.md (hunk 6 above)
Comment (carofun) [reply git.md]: Changes have been done.

File: git.md (hunk 8 above)
Comment (carofun) [reply git.md]: Changes have been done.

**Review by jmaupetit — APPROVED** [review]

**Pull request merged**